libraries/chain_kv/include/b1/session/rocks_session.hpp

```diff
@@ -54,6 +59,7 @@ class session<rocksdb_t> {
       bool            operator!=(const rocks_iterator& other) const;
 
       shared_bytes key() const;
+      bool         deleted() const;
```

Comment:
I think putting definitions within the class declaration cause a lot of clutter and makes it harder to see what the definition of the type is.  Technically I could just move the definitions into a source file so if/when I get around to that it would be much simpler to do having this split between definition and declaration.